dev-docs/modules/paapi.md

```diff
@@ -0,0 +1,171 @@
+title: Module - PAAPI
+description: Protected audience API
+module_code : paapi
+display_name : PAAPI
+enable_download : true
+sidebarType : 1
+---
+
+# PAAPI module
+
+This module allows Prebid.js to support PAAPI, formerly known as [FLEDGE](https://github.com/WICG/turtledove/blob/main/FLEDGE.md).
+
+This document covers the steps necessary for publishers to enable PAAPI on their inventory. It also describes
+the changes Bid Adapters need to implement in order to support PAAPI.
+
+A related module, [fledgeForGpt](/dev-docs/modules/fledgeForGpt.html), adds support specifically for GPT's [experimental FLEDGE
+support](https://github.com/google/ads-privacy/tree/master/proposals/fledge-multiple-seller-testing).
+
+## Publisher Integration
+
+To use PAAPI, publishers must:
+
+- include this module in their Prebid.js bundle
+
+   ```bash
+   gulp build --modules=paapi,...
+   ```
+
+- enable PAAPI, globally or by ad unit, through [configuration](#config)
+- manage the PAAPI auctions. This can be delegated to GPT with the [fledgeForGpt module](/dev-docs/modules/fledgeForGpt.html); homegrown solutions are possible with [getPAAPIConfig](/dev-docs/publisher-api-reference/getPAAPIConfig.html), but out of scope for this document.
+
+<a id="config"></a>
+
+### Module Configuration
+
+This module exposes the following settings:
+
+{: .table .table-bordered .table-striped }
+|Name |Type |Description |Notes |
+| ------------ | ------------ | ------------ |------------ |
+|enabled | Boolean |Enable/disable the module |Defaults to `false` |
+|bidders | Array[String] |Optional list of bidders |Defaults to all bidders |
+|defaultForSlots | Number |Default value for `imp.ext.ae` in requests for specified bidders |Should be 1 |
+
+As noted above, PAAPI support is disabled by default. To enable it, set the `enabled` value to `true` for this module and configure `defaultForSlots` to be `1` (meaning _Client-side auction_).
+using the `setConfig` method of Prebid.js. Optionally, a list of
+bidders to apply these settings to may be provided:
+
+```js
+pbjs.que.push(function() {
+  pbjs.setConfig({
+    paapi: {
+      enabled: true,
+      bidders: ['openx', 'rtbhouse'],
+      defaultForSlots: 1
+    }
+  });
+});
+```
+
+### Bidder Configuration
+
+This module adds the following setting for bidders:
+
+{: .table .table-bordered .table-striped }
+|Name |Type |Description |Notes |
+| ------------ | ------------ | ------------ |------------ |
+| fledgeEnabled | Boolean | Enable/disable a bidder to participate in FLEDGE | Defaults to `false` |
+|defaultForSlots | Number |Default value for `imp.ext.ae` in requests for specified bidders |Should be 1|
+
+In addition to enabling PAAPI at the module level, individual bidders must also be enabled. This allows publishers to
+selectively test with one or more bidders as they desire. To enable one or more bidders, use the `setBidderConfig` method
+of Prebid.js:
+
+```js
+pbjs.setBidderConfig({
+    bidders: ["openx"],
```

Comment:
question: should this be a generic bidder adapter name, like we have for other parameters?